tests/unit/test_adapter.py

```diff
@@ -195,10 +198,11 @@ def test_invoke_failure_no_catalog(self, mock_process_items,
 
         with self.assertRaises(ZarrException) as context_manager:
             harmony_adapter.invoke()
-            self.assertEqual(
-                str(context_manager.exception),
-                'Invoking NetCDF-to-Zarr without STAC catalog is not supported.'
-            )
+
+        self.assertEqual(
+            context_manager.exception.message,
+            'Request failed due to an incorrect service workflow'
```

Comment:
Did something in the test change so that we're not testing this line anymore? https://github.com/nasa/harmony-netcdf-to-zarr/blob/DAS-1536-assertRaises/harmony_netcdf_to_zarr/adapter.py#L71 
Maybe the test wasn't set up properly before?  Is it possible to test that condition also?

Reply:
Good question - I'll take a closer look as to why the wrong exception was being raised. I'd guess one of the properties being looked for in the first condition isn't there.

Reply:
Yeah - the message being used didn't have the `Message.format.mime` property set, so it was hitting the first condition, rather than the second.